src/components/Orders/OrderItem/OrderItem.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './OrderItem.scss';
+
+const OrderItem = (props) => {
+
+   const { product } = props;
+
+   return (
+      <tr>
+         <td>{product.Brand}</td>
+         <td>{product.DeviceName}</td>
+         <td>{product.announced}</td>
+         <td>{product.price}</td>
+      </tr>
+   )
+}
+
+OrderItem.propTypes = {
+   product: PropTypes.object,
+   Brand: PropTypes.string,
+   DeviceName: PropTypes.string,
+   announced: PropTypes.string,
+   Price: PropTypes.string
```

Comment:
Those types are not correct?!!!!! 
```js
const { product, Brand, DeviceName, announced, Price } = props;
```
I think this is not the desired props.
Try this approach:
```js
OrderItem.propTypes = {
product: PropTypes.exact({
    Brand: PropTypes.string.isRequired,
   DeviceName: PropTypes.string.isRequired,
   announced: PropTypes.string.isRequired,
   Price: PropTypes.string.isRequired
  }).isRequired
}
```
or check https://www.npmjs.com/package/prop-types docs for other examples